src/processout/processout.ts

```diff
@@ -682,6 +714,30 @@ module ProcessOut {
             }.bind(this);
         }
 
+        /**
+         * getCustomerTokenActionURL returns the customer token action URL
+         * @param {string} customerID
+         * @param {string} tokenID
+         * @param {any|string} gatewayConf 
+         * @return {string}
+         */
+        public getCustomerTokenActionURL(
+            customerID:  string,
+            tokenID:     string,
+            gatewayConf: any
+        ): string {
+            var gatewayConfID = gatewayConf;
+            if (gatewayConf && gatewayConf.id) {
+                gatewayConfID = gatewayConf.id;
+            }
+
+            if (!customerID || !tokenID) {
+                throw new Exception("processout-js.missing-resource-id", "A Customer Token Action was requested, but a customer ID or a token ID was missing. Make sure you didn't rather want to call `invoice` helpers on the gateway configuration instead of `customerToken` ones.")
```

Comment:
```suggestion
                throw new Exception("processout-js.missing-resource-id", "A Customer Token Action was requested, but a customer ID or a token ID was missing. Make sure you did not rather want to call the `invoice` helpers on the gateway configuration instead of the `customerToken` ones.")
```

nit